test_scripts/API/AddCommand/ATF_AddCommand_UNSUP_RES_to_UI_AddCmd_WARNINGS_to_VR_AddCmd.lua

```diff
@@ -0,0 +1,134 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this particular test it is checked case when VR.AddCommand gets Warnings and UI.AddCommand gets UNSUPPORTED_RESOURCE from HMI
+-- 1. Used preconditions: App is activated and registered SUCCESSFULLY
+-- 2. Performed steps: 
+-- MOB -> SDL: sends AddCommand
+-- SDL -> HMI: resends VR.AddCommand and UI.AddCommand
+-- HMI -> SDL: VR.AddCommand (WARNINGS), UI.AddCommand (UNSUPPORTED_RESOURCE)
+--
+-- Expected result:
+-- SDL -> MOB: AddCommand (result code: UNSUPPORTED_RESOURCE, success: true)
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.SDLStoragePath = config.pathToSDL .. "storage/"
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.. "/"
+local ServerAddress = commonFunctions:read_parameter_from_smart_device_link_ini("ServerAddress") 
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
```

Comment:
@istoimenova , Please see updated in commit e3c195283c2acd34494b065de5dde8270b56f351